ssl/statem/extensions.c

```diff
@@ -1242,6 +1242,13 @@ int tls_psk_do_binder(SSL *s, const EVP_MD *md, const unsigned char *msgstart,
     const char *label;
     size_t bindersize, labelsize, hashsize = EVP_MD_size(md);
     int ret = -1;
+    int usepskfored = 0;
+
+    if (external
+            && s->early_data_state == SSL_EARLY_DATA_CONNECTING
+            && s->session->ext.max_early_data == 0
```

Comment:
What does the check s->session->ext.max_early_data == 0 do?
I guess it relates to having both an s->session and a s->psksession but don't fully undestand how that interacts yet.